pkg/controller/direct/workflows/mapper.validation.txt

```diff
@@ -0,0 +1 @@
+[mismatch_ref] Kind=WorkflowsWorkflowSpec, Version=v1alpha1, KRM field "KMSCryptoKeyRef" has no matching proto field found (make sure the KRM field is the pascal case of the proto with "Ref" suffix)
```

Comment:
This is fixed in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/5304